util/file.go

```diff
@@ -165,3 +168,23 @@ func ReadLinesFromFile(filename string) ([]string, error) {
 
 	return lines, nil
 }
+
+func ReadJsonFile(filename string) (string, error) {
```

Comment:
I don't see any evidence of this function being used. Is this PR complete or is it intended to just be used in consuming projects?

Reply:
This is for json file input support in seed-cli